content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -54,6 +54,23 @@ stateful tasks.
 
 {{< figure src="/images/docs/scheduling-framework-extensions.png" title="scheduling framework extension points" class="diagram-large">}}
 
+### PreEnqueue {#pre-enqueue}
+
+These plugins are called prior to adding Pods to the internal active queue, where Pods are marked as
+ready for scheduling.
+
+Only when all PreEnqueue plugins return `Success`, the Pod is allowed to enter the active queue.
+Otherwise, it's placed in the internal unschedulable Pods list.
+
+{{< note >}}
+
+More details about how internal scheduler queues work, please check this
+[document](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-scheduling/scheduler_queues.md).
+
+{{< /note >}}
+
+> Check more details
```

Comment:
```suggestion
For more details about how internal scheduler queues work, read
[Scheduling queue in kube-scheduler](https://github.com/kubernetes/community/blob/f03b6d5692bd979f07dd472e7b6836b2dad0fd9b/contributors/devel/sig-scheduling/scheduler_queues.md).
```
I used a commit hash for permanence. If that linked document changes, we should update the hyperlink.

Aside: we could move that document to the reference docs or to https://k8s.dev/

Reply:
Updated.